### PR TITLE
Fix current compilation error

### PR DESCRIPTION
**Pull request by wind57**

### CODE REVIEW

File: spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configserver/src/test/java/org/springframework/cloud/kubernetes/configserver/KubernetesEnvironmentRepositoryTests.java [hunk 1]
```diff
@@ -264,25 +264,32 @@ else if (propertySource.getName().equals("secret.stores.stores-dev.default")) {
...
+				eq(null), eq(null), eq(null), eq(null))).thenReturn(CONFIGMAP_DEFAULT_LIST);
 		when(coreApi.listNamespacedSecret(eq("default"), eq(null), eq(null), eq(null), eq(null), eq(null), eq(null),
-			eq(null), eq(null), eq(null), eq(null))).thenReturn(SECRET_LIST);
+				eq(null), eq(null), eq(null), eq(null))).thenReturn(SECRET_LIST);
 		when(coreApi.listNamespacedConfigMap(eq("dev"), eq(null), eq(null), eq(null), eq(null), eq(null), eq(null),
-			eq(null), eq(null), eq(null), eq(null))).thenReturn(CONFIGMAP_DEV_LIST);
+				eq(null), eq(null), eq(null), eq(null))).thenReturn(CONFIGMAP_DEV_LIST);
 		KubernetesEnvironmentRepository environmentRepository = new KubernetesEnvironmentRepository(coreApi,
-			kubernetesPropertySourceSuppliers, "default");
+				KUBERNETES_PROPERTY_SOURCE_SUPPLIER, "default");
```
Comment (wind57) [inline spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configserver/src/test/java/org/springframework/cloud/kubernetes/configserver/KubernetesEnvironmentRepositoryTests.java:273]: `KUBERNETES_PROPERTY_SOURCE_SUPPLIER` this was missing, probably a small merge issue. The rest is formatting, nothing else

**Comment by ryanjbaxter** [comment]
Thanks, I merged some stuff forward and messed it all up.  I should not have done this on a Friday afternoon.

**Comment by wind57** [comment]
debatable if you should not have done it (joking, of course) :) I was very eager to get some of these small PRs in, before I present some really meaningful work to you. Thank you for merging it

**Comment by ryanjbaxter** [comment]
😂 it's not debatable I definitely shouldn't have done it.

Thanks again!

**Pull request merged**